src/App.tsx

```diff
@@ -133,9 +133,27 @@ const TMP_DATA: MarkerFileData = {
 	}]
 };
 
-export default function App() {
+    export default function App() {
+	const [openFileSelector, {filesContent, loading}] = useFilePicker({
+		accept: '.txt',
+	});
+
+	if (loading) {
```

Comment:
Do we really want to hide the entire UI whenever a file is being read?

Reply:
This was also temporary as part of the learning process. It was last minute and needed to be closed, but has since been refactored.